config/kubernetes/base/operator.yaml

```diff
@@ -63,3 +68,26 @@ spec:
             configMapKeyRef:
               name: tekton-config-defaults
               key: DEFAULT_TARGET_NAMESPACE
+      - name: tekton-operator-cluster-operations
```

Comment:
should we add a comment to be more precise ?

Reply:
i was thinking the long name will be sufficient 🧑‍💻 
but i could thinking like that because i was working with it.

i shall add a coment 👍

Reply:
Yup for those who is new to this change it will be bit difficult to get the meaning